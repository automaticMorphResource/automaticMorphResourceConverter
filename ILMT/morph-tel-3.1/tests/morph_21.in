1	aMxuke	unk	
1	palleVllo	unk	
1	cAlA	unk	
1	maMxi	unk	
1	ArogyaMgA	unk	
1	uMtAru	unk	
1	.	unk	
