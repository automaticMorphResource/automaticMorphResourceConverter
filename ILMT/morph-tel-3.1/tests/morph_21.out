1	aMxuke	unk	<fs af='aMxuke,avy,,,,,,'>
2	palleVllo	unk	<fs af='palleV,n,,pl,,,lo,lo'>
3	cAlA	unk	<fs af='cAlA,avy,,,,,0,0_avy'>|<fs af='cAlu,v,any,any,any,,i_A,i_A'>|<fs af='cAlu,avy,,,,,A,A_avy'>
4	maMxi	unk	<fs af='maMxi,n,,pl,,d,0,0'>|<fs af='maMxi,n,,pl,,,vu,vu'>|<fs af='maMxi,n,,pl,,o,ti,ti'>|<fs af='manu,v,fn,sg,3,,A,A'>
5	ArogyaMgA	unk	<fs af='ArogyaM,n,,sg,,,gA,gA'>
6	uMtAru	unk	<fs af='uMdu,v,fm,pl,3,,wA,wA'>
7	.	unk	<fs af='&dot;,punc,,,,,,'>
